staging/src/k8s.io/legacy-cloud-providers/azure/azure_standard.go

```diff
@@ -37,9 +37,15 @@ import (
 	"k8s.io/apimachinery/pkg/util/uuid"
 	cloudprovider "k8s.io/cloud-provider"
 	"k8s.io/klog"
+
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/component-base/featuregate"
+	utilnet "k8s.io/utils/net"
 )
 
 const (
+	IPv6DualStack featuregate.Feature = "IPv6DualStack"
```

Comment:
consider using `features.IPv6DualStack` from `pkg/features` directly here?

Reply:
We can not vendor `pkg/features` here, it will create a circular dependency